src/net/connection.rs

```diff
@@ -457,15 +457,15 @@ impl std::default::Default for ConnectionOptions {
             public_ip_max_retries: 3, // maximum number of retries before self-throttling for $public_ip_timeout
             max_block_push: 10, // maximum number of blocksData messages to push out via our anti-entropy protocol
             max_microblock_push: 10, // maximum number of microblocks messages to push out via our anti-entropy protocol
-            antientropy_retry: 3600, // retry pushing data only once every hour
+            antientropy_retry: 60,   // retry pushing data once every minute
             antientropy_public: true, // run antientropy even if we're NOT NAT'ed
             max_buffered_blocks_available: 1,
             max_buffered_microblocks_available: 1,
             max_buffered_blocks: 1,
             max_buffered_microblocks: 10,
             mempool_sync_interval: 30, // number of seconds in-between mempool sync
-            mempool_max_tx_query: MAX_BLOOM_COUNTER_TXS.into(),
-            mempool_sync_timeout: 60, // how long a mempool sync can go for
+            mempool_max_tx_query: 128, // maximum number of transactions to return
```

Comment:
This used to be `MAX_BLOOM_COUNTER_TXS`, is that constant no longer used anywhere? Should it be updated to 128?

Reply:
Yes; this constant is used in the mempool bloom filter implementation.

This number controls how many transactions will be _visited_ in the DB, not returned.  I'll update the field member.